Dockerfile.template

```diff
@@ -0,0 +1,45 @@
+FROM node:%%NODE_VERSION%%-alpine
```

Comment:
Apparently official images manage all their variants with this token scheme and simple sed and cp commands. So I did that here.